addons/im_livechat/static/src/embed/core/thread_model_patch.js

```diff
@@ -21,7 +21,7 @@ patch(Thread, {
                     livechatService.updateSession({
                         state: thread.state,
                         seen_message_id: thread.seen_message_id,
-                        channel: thread.channel,
+                        channel: thread,
```

Comment:
Won't this add to the session several values that are not necessary?

Reply:
We don't care, the session cookie conceptually stores livechat data, so in modelling terms it should store thread. That's what we care